src/map/map.ts

```diff
@@ -214,6 +214,12 @@ export abstract class BaseMap extends Events implements BaseMapDefinition {
         this.leafletInstance.on("click", (evt: L.LeafletMouseEvent) =>
             this.handleMapClick(evt)
         );
+        this.on('rendered', () => {
+            if (this.options.zoomMarkers) {
+                this.log(`Zooming to markers.`);
+                this.zoomAllMarkers();
+            }
+        })
```

Comment:
This could also live inside of the `render` function, but this felt like a more appropriate place, alongside the other event handlers. If you I don't see where the `rendered` event is handled elsewhere, so if you wanted this to be a stable external-facing event, then maybe relocating this might make sense.